backend/src/test/java/com/allog/dallog/categoryrole/application/CategoryRoleServiceTest.java

```diff
@@ -32,220 +31,247 @@
-                    actual.getSubscribers().stream().map(it -> it.getMember().getDisplayName())
-                            .collect(Collectors.toList()))
-                    .containsExactly(관리자_이름, 후디_이름, 리버_이름);
-        });
+        assertThat(actual.getSubscribers().size()).isEqualTo(2);
     }
 
-    @DisplayName("특정 카테고리의 구독자 목록을 ADMIN이 아닌 회원이 호출하면 예외가 발생한다.")
     @Test
-    void 특정_카테고리의_구독자_목록을_ADMIN이_아닌_회원이_호출하면_예외가_발생한다() {
+    void 관리자_권한이_아닌_회원이_카테고리_구독자_목록을_조회하면_예외가_발생한다() {
         // given
-        Member 관리자 = memberRepository.save(관리자());
-        Member 후디 = memberRepository.save(후디());
+        GivenBuilder 나인 = 나인().회원_가입을_한다(나인_이메일, 나인_이름, 나인_프로필_URL)
+                .카테고리를_생성한다(취업_카테고리_이름, NORMAL);
 
-        CategoryResponse 카테고리 = categoryService.save(관리자.getId(), 공통_일정_생성_요청);
-        subscriptionService.save(후디.getId(), 카테고리.getId());
+        GivenBuilder 티거 = 티거().회원_가입을_한다(티거_이메일, 티거_이름, 티거_프로필_URL)
+                .카테고리를_구독한다(나인.카테고리());
 
         // when & then
-        assertThatThrownBy(() -> categoryRoleService.findSubscribers(후디.getId(), 카테고리.getId()))
+        assertThatThrownBy(() -> categoryRoleService.findSubscribers(티거.회원().getId(), 나인.카테고리().getId()))
                 .isInstanceOf(NoCategoryAuthorityException.class);
     }
 
-    @DisplayName("관리자(NONE 이상) 역할로 변경하려는 대상이 이미 50개 이상의 카테고리에서 관리자로 참여하고 있는 경우 예외가 발생한다.")
     @Test
-    void 관리자_역할로_변경하려는_대상이_이미_50개_이상의_카테고리에서_관리자로_참여하고_있는_경우_예외가_발생한다() {
+    void 관리자_역할로_변경하려는_회원이_이미_50개_이상의_카테고리에_관리자_권한이_있으면_예외가_발생한다() {
         // given
-        Member 관리자 = memberRepository.save(관리자());
-        CategoryResponse 공통_일정 = categoryService.save(관리자.getId(), 공통_일정_생성_요청);
+        GivenBuilder 나인 = 나인().회원_가입을_한다(나인_이메일, 나인_이름, 나인_프로필_URL)
+                .카테고리를_생성한다(취업_카테고리_이름, NORMAL);
 
-        Member 후디 = memberRepository.save(후디());
+        GivenBuilder 티거 = 티거().회원_가입을_한다(티거_이메일, 티거_이름, 티거_프로필_URL)
+                .카테고리를_구독한다(나인.카테고리());
         for (int i = 0; i < 50; i++) {
-            CategoryCreateRequest request = new CategoryCreateRequest("카테고리 " + i, CategoryType.NORMAL);
-            categoryService.save(후디.getId(), request);
-        } // 후디는 이미 50개의 카테고리를 관리하고 있음
-
-        subscriptionService.save(후디.getId(), 공통_일정.getId()); // 후디가 공통 일정을 구독함
+            티거.카테고리를_생성한다("카테고리 " + i, NORMAL);
+        }
 
-        assertThatThrownBy(() -> categoryRoleService.updateRole(관리자.getId(), 후디.getId(), 공통_일정.getId(),
-                new CategoryRoleUpdateRequest(ADMIN)))
+        // when & then
+        assertThatThrownBy(
+                () -> categoryRoleService.updateRole(
+                        나인.회원().getId(), 티거.회원().getId(), 나인.카테고리().getId(), 카테고리_관리권한_부여_요청))
                 .isInstanceOf(ManagingCategoryLimitExcessException.class);
     }
 
     @Transactional
-    @DisplayName("ADMIN 회원이 구독자의 역할을 변경할 수 있다")
     @Test
-    void ADMIN_회원이_구독자의_역할을_변경할_수_있다() {
+    void 카테고리_관리권한이_관리자인_회원이_다른_관리자의_권한을_변경한다() {
```

Comment:
```suggestion
    void 카테고리_권한이_관리자인_회원이_다른_관리자의_권한을_변경한다() {
```
r: 카테고리 관리권한 -> 카테고리 권한으로 통일하는 건 어떨까요?

Reply:
수정해보았어요!